lib/atlas/exporter/essential_exporter.rb

```diff
@@ -31,11 +32,21 @@ def nodes_hash(nodes)
     #
     # Returns a Hash.
     def edges_hash(edges)
-      edges.each_with_object({}) do |edge, hash|
+      data = edges.each_with_object({}) do |edge, hash|
         edge_key = edge.properties[:model].key
 
         hash[edge_key] = edge.properties.except(:model)
       end
+
+      # Yay coupling carrier special cases!
```

Comment:
Do we really need that here?
This is already done when doing the MsgPack-FullExport later. So once the graph is loaded in ETEngine it is gonna be done anyways. For creating the graph when importing the YAML-EssentialExport it should not be necessary since it is also not necessary for loading a normal FullDataset.